vm/src/stdlib/itertools.rs

```diff
@@ -1,5 +1,14 @@
 pub(crate) use decl::make_module;
 
+macro_rules! handle_pyiter_return {
```

Comment:
This macro is expected to be defined on `vm/src/protocol/iter.rs` because it is mainly a function of PyIterReturn.

The name can be simplified as what it means in Python. 
```suggestion
macro_rules! raise_stop {
```

I like this idea 👍